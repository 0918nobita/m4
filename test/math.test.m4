include(math.m4)dnl
add(2, 3, 4)
sub(10, 1, 2)
[fact(3)] = fact(3)
[fact(4)] = fact(4)
[fact(5)] = fact(5)
