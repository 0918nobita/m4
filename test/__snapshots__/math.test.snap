9
7
fact(3) = 6
fact(4) = 24
fact(5) = 120
